src/request_handler.cc

```diff
@@ -48,22 +48,24 @@ RequestHandler::RequestHandler(std::shared_ptr<ContentManager> content)
 {
 }
 
-void RequestHandler::splitUrl(const std::string& url, char separator, std::string& path, std::string& parameters)
+std::tuple<std::string_view, std::string_view> RequestHandler::splitUrl(std::string_view url, char separator)
```

Comment:
pair makes more sense here. tuple is used nowhere in the code.

Reply:
I'm OK with either version.